e2e/nx-run/src/run.test.ts

```diff
@@ -466,6 +465,7 @@ describe('Nx Running Tests', () => {
           version: '0.0.1',
           scripts: {
             build: 'nx exec -- echo HELLO',
+            'build:option': 'nx exec -- echo HELLO WORLD',
```

Comment:
Can this echo something other than `HELLO WORLD`, there is another test that is checking the `build` script accepts misc flags that checks `HELLO WORLD` that I could see getting confused here.